Package.swift

```diff
@@ -20,14 +20,12 @@ let package = Package(
 	dependencies: [
 		// Dependencies declare other packages that this package depends on.
 		// .package(url: /* package url */, from: "1.0.0"),
-
 		.package(url: "https://github.com/GigaBitcoin/secp256k1.swift.git", exact: "0.10.0"),
 		.package(url: "https://github.com/argentlabs/web3.swift", from: "1.1.0"),
 		.package(url: "https://github.com/1024jp/GzipSwift", from: "5.2.0"),
 		.package(url: "https://github.com/bufbuild/connect-swift", exact: "0.3.0"),
 		.package(url: "https://github.com/apple/swift-docc-plugin.git", from: "1.0.0"),
-		.package(url: "https://github.com/xmtp/libxmtp-swift", revision: "92274fe"),
-//		.package(path: "../libxmtp-swift")
+		.package(url: "https://github.com/xmtp/libxmtp-swift", revision: "1d7068f181a402b2cd8b9520ccb768e65a55ec32"),
```

Comment:
I tagged it on main branch at [503086d](https://github.com/xmtp/libxmtp-swift/commit/503086d91ba6c3420aca118e4812d62a004d17ee), after merging your PR, so you'll prob want to update to that commit
```suggestion
		.package(url: "https://github.com/xmtp/libxmtp-swift", revision: "503086d"),
```